external/README.md

```diff
@@ -2,6 +2,25 @@
 
 Third Party container tests help verify that the adoptium binaries are *good* by running a variety of Java applications inside of Docker containers. adoptium/aqa-tests/[Issue #172](https://github.com/adoptium/aqa-tests/issues/172) lists the applications that we have initially targeted to best exercise the adoptium binaries.  For each application, we choose to run a selection of their functional tests.
```

Comment:
> by running a variety of Java applications inside of Docker containers

As Podman is available, would be better to update as by running a variety of Java applications inside of **Docker|Podman** containers

Reply:
There is a lot of "Docker" keyword in a whole adoptium tests, tools and build. I 'm changing it where it is reasonable, but to do a mass sed, would make it extremly messy. I will check The readme and see waht can be improved. THAnx!

Reply:
Fixed. Tahnx a lot, readme now looks much better